spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1693,7 +1693,10 @@ properties (with the prefix stripped) when the local `EntityManagerFactory` is c
 
 [[howto-configure-hibernate-naming-strategy]]
 === Configure Hibernate Naming Strategy
-Hibernate defines `Physical` and `Implicit` naming strategies. Spring Boot configures
+Since version 5, Hibernate defines `Physical` and `Implicit` naming strategies which you
+can configure in Spring Boot by providing a fully qualified class name to the properties
+`spring.jpa.hibernate.naming.physical-strategy` and
+`spring.jpa.hibernate.naming.implicit-strategy`. Spring Boot configures
```

Comment:
If only one of two properties is explicitly mentioned in the section, I feel uncertain about the second property (at least until I scroll down to the appendix, which I wasn't aware of at first). To remove that uncertainty, I would have liked both properties be named explicitly.

Might just be me, but